ui/lib/core/addon/components/menu-loader.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @module MenuLoader
+ * MenuLoader components are used to...
```

Comment:
this description should probably be updated even if it doesn't go into storybook right away. :)